tests/aws/services/stepfunctions/v2/express/test_express_async.snapshot.json

```diff
@@ -0,0 +1,218 @@
+                  "Input": {
+                    "message": "HelloWorld!"
+                  },
+                  "Name": "<ExpressExecArn_Part1_0idx>",
+                  "RoleArn": "sfn_role_arn",
+                  "StartTime": "start-time",
+                  "RedriveCount": 0
+                },
+                "StateMachine": {
+                  "Id": "arn:aws:states:<region>:111111111111:stateMachine:<ArnPart_0idx>",
+                  "Name": "<ArnPart_0idx>"
+                },
+                "State": {
+                  "Name": "QueryValues1",
+                  "EnteredTime": "entered-time"
+                }
+              },
+              "statemachine_name": "<ArnPart_0idx>"
+            },
+            "query_values_2": {
+              "context_object": {
+                "Execution": {
+                  "Id": "arn:aws:states:<region>:111111111111:express:<ArnPart_0idx>:<ExpressExecArn_Part1_0idx>:<ExpressExecArn_Part2_0idx>",
+                  "Input": {
+                    "message": "HelloWorld!"
+                  },
+                  "Name": "<ExpressExecArn_Part1_0idx>",
+                  "RoleArn": "sfn_role_arn",
+                  "StartTime": "start-time",
+                  "RedriveCount": 0
+                },
+                "StateMachine": {
+                  "Id": "arn:aws:states:<region>:111111111111:stateMachine:<ArnPart_0idx>",
+                  "Name": "<ArnPart_0idx>"
+                },
+                "State": {
+                  "Name": "QueryValues2",
+                  "EnteredTime": "entered-time"
+                }
+              }
+            }
+          },
+          "outputDetails": {
+            "truncated": false
+          }
+        },
+        "event_timestamp": "timestamp",
+        "execution_arn": "arn:aws:states:<region>:111111111111:express:<ArnPart_0idx>:<ExpressExecArn_Part1_0idx>:<ExpressExecArn_Part2_0idx>",
+        "id": "6",
+        "previous_event_id": "5",
+        "redrive_count": "0",
+        "type": "ExecutionSucceeded"
+      }
+    }
+  },
+  "tests/aws/services/stepfunctions/v2/express/test_express_async.py::TestExpressAsync::test_catch": {
+    "recorded-date": "03-07-2024, 16:33:03",
+    "recorded-content": {
+      "creation_response": {
+        "creationDate": "datetime",
+        "stateMachineArn": "arn:aws:states:<region>:111111111111:stateMachine:<ArnPart_0idx>",
+        "ResponseMetadata": {
+          "HTTPHeaders": {},
+          "HTTPStatusCode": 200
+        }
+      },
+      "end_event": {
+        "details": {
+          "output": {
+            "Error": "Exception",
+            "Cause": "{\"errorMessage\":\"Some exception was raised.\",\"errorType\":\"Exception\",\"requestId\":\"30ebd6a2-7614-461d-a18f-61af2149852c\",\"stackTrace\":[\"  File \\\"/var/task/handler.py\\\", line 2, in handler\\n    raise Exception(\\\"Some exception was raised.\\\")\\n\"]}"
```

Comment:
The `requestId` should probably be transformed if we ever want to match the cause here